### PR TITLE
fix: spelling errors in error responses

### PR DESCRIPTION
**Pull request by gordsport**
supersedes the following PR:

- https://github.com/vegaprotocol/vega/pull/7487

### CODE REVIEW

File: core/evtforward/ethereum/filterer.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ func NewLogFilterer(
 
 	collateralBridgeFilterer, err := bridge.NewErc20BridgeLogicRestrictedFilterer(collateralBridge.Address(), ethClient)
 	if err != nil {
-		return nil, fmt.Errorf("couldn't create log filterer for ERC20 brigde: %w", err)
+		return nil, fmt.Errorf("couldn't create log filterer for ERC20 bridge: %w", err)
```
Comment (vega-paul) [inline core/evtforward/ethereum/filterer.go:111]: Filterer>filter

File: core/evtforward/ethereum/filterer.go [hunk 2]
```diff
@@ -118,7 +118,7 @@ func NewLogFilterer(
 
 	stakingBridgeFilterer, err := staking.NewStakingFilterer(stakingBridge.Address(), ethClient)
 	if err != nil {
-		return nil, fmt.Errorf("couldn't create log filterer for staking brigde: %w", err)
+		return nil, fmt.Errorf("couldn't create log filterer for staking bridge: %w", err)
```
Comment (vega-paul) [inline core/evtforward/ethereum/filterer.go:121]: Filterer>filter

File: core/evtforward/ethereum/filterer.go [hunk 3]
```diff
@@ -128,7 +128,7 @@ func NewLogFilterer(
 
 	vestingBridgeFilterer, err := staking.NewStakingFilterer(vestingBridge.Address(), ethClient)
 	if err != nil {
-		return nil, fmt.Errorf("couldn't create log filterer for vesting brigde: %w", err)
+		return nil, fmt.Errorf("couldn't create log filterer for vesting bridge: %w", err)
```
Comment (vega-paul) [inline core/evtforward/ethereum/filterer.go:131]: Filterer>filter

**Review by vega-paul — DISMISSED** [review]

**Comment by gordsport** [comment]
@jeremyletang @ValentinTrinque 

Looks like you guys have edited/moved this file. Please can you advise on Paul comment above?

File: core/evtforward/ethereum/filterer.go (hunk 1 above)
Comment (jeremyletang) [reply core/evtforward/ethereum/filterer.go:111]: nope filterer is actually the right term here.

**Review by jeremyletang — APPROVED** [review]

File: core/evtforward/ethereum/filterer.go (hunk 2 above)
Comment (gordsport) [reply core/evtforward/ethereum/filterer.go:121]: https://github.com/vegaprotocol/vega/pull/7739/files#r1122911952

File: core/evtforward/ethereum/filterer.go (hunk 3 above)
Comment (gordsport) [reply core/evtforward/ethereum/filterer.go:131]: https://github.com/vegaprotocol/vega/pull/7739/files#r1122911952

**Pull request merged**